web/thesauruses/kotlin/1.5/io.json

```diff
@@ -0,0 +1,270 @@
+{
+    "meta": {
+        "language": "kotlin",
+        "language_name": "Kotlin",
+        "structure": "io",
+        "language_version": "1.5"
+    },
+    "concepts": {
+    "write_line": {
+        "code": [
+        "print(\"...\")"
+        ]
+    },
+    "write_line_with_new_line": {
+        "code": [
+        "println(\"...\")"
+        ]
+    },
+    "read_line": {
+        "code": [
+        "val variable_name = readln()"
+        ]
+    },
+    "read_char": {
+        "code": [
+        "val variable_name = readln()",
+        "val first_char = variable_name[0]"
+        ],
+        "comment": [
+        "First character can be read by accessing the first element of the string"
+        ]
```

Comment:
Generally I've tried to keep the best practice that if you're writing code to replicate a function that doesn't exist, it shouldn't be in here. So if there's no built-in way to do that, it should probably be `not-implemented: true`

Reply:
Has been changed to:

![image](https://github.com/user-attachments/assets/810e3356-6fb0-4391-97e5-dec1ba8f810e)